### DISCUSSION TITLE
antidos version 2

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#362] [Show and tell]
added whitelist and automatic ban lifting
https://github.com/rejetto/antidos

**Comment by rejetto** [comment]
in version 2.3, whitelist supports masks

**Comment by SimonChiu930** [comment]
Cannot read properties of undefined (reading 'makeNetMatcher')

**Reply by rejetto** [reply]
thanks for reporting, i just published the fix.
Please update again
